packages/zoe/src/contracts/autoswap.js

```diff
@@ -209,14 +239,35 @@ export const makeContract = harden((zoe, terms) => {
           /**
            * `getPrice` calculates the result of a trade, given a certain amount
            * of digital assets in.
-           * @param {amount} amountIn - the amount of digital assets to be sent in
+           * @param {object} amountInObj - the amount of digital
+           * assets to be sent in, keyed by keyword
            */
-          getPrice: amountIn => getPrice(getPoolAmounts(), amountIn).amountOut,
+          getPrice: amountInObj => {
+            const [inKeyword] = Object.getOwnPropertyNames(amountInObj);
```

Comment:
If `amountInObj` has multiple properties, this will silently select one and ignore the others.